### PR TITLE
🐛Fix: v-bind order in `vue/attribute-order` rule

### LINKED ISSUE #410: v-bind attribute should be sorted by its real name
In new rule `vue/attributes-order`, **v-bind** has an order as same as v-model, but I think it should not be regarded as a single group. For example **v-bind:is** should be placed at first but not together with other attributes.

### LINKED ISSUE #422: vue/attributes-order: v-dom-portal errors if placed after v-if
**Tell us about your environment**

* **ESLint Version: 4.14.0**
* **eslint-plugin-vue Version: 4.3.0**
* **Node Version: 8.9.4**

**Please show your full configuration:**

```
"vue/attributes-order": [2, {
          order: [
              "DEFINITION",
              "LIST_RENDERING",
              "CONDITIONALS",
              "RENDER_MODIFIERS",
              "GLOBAL",
              "UNIQUE",
              "CONTENT",
              "OTHER_ATTR",
              "BINDING",
              "EVENTS"
          ]
      }]

```

**What did you do? Please include the actual source code causing the issue.**

```vue
<div v-if="isVisible" v-dom-portal="true" :class="classes"></div>
```

**What did you expect to happen?**
v-dom-portal directive should not fail linting, as it should be OTHER_ATTR imo

**What actually happened? Please include the actual, raw output from ESLint.**
Instead it seems to be recognised as either DEFINITION or LIST_RENDERING, and the following message displays:

```
Attribute "v-dom-portal" should go before "v-if". (vue/attributes-order)
```

### PR DESCRIPTION
**Pull request by ydfzgyj**
close #410 , close #422

### CODE REVIEW

File: docs/rules/attributes-order.md [hunk 1]
```diff
@@ -17,10 +17,10 @@ ex: 'v-once', 'v-pre'
 ex: 'id'
 - UNIQUE
 ex: 'ref', 'key', 'slot'
-- BINDING
-ex: 'v-model', 'v-bind', ':property="foo"'
+- TWO\_WAY\_BINDING
+ex: 'v-model'
 - OTHER_ATTR
-ex: 'customProp="foo"'
+ex: 'custom-prop="foo"', 'v-bind:prop="foo"', ':prop="foo"', 'v-custom-directive'
```
Comment (michalsnik) [inline docs/rules/attributes-order.md]: I don't think we should mix other attributes with bindings.
Let's keep both groups: `TWO_WAY_BINDINGS` and `BINDINGS`

Comment (ydfzgyj) [reply docs/rules/attributes-order.md]: It's a common situation that when user needs to change a static property value to a dynamic binding value. If we separate them into two groups, the user may have to modify the order of these properties and feel confused. Maybe it's still better to let user themselves to sort these attributes.

**Comment by ydfzgyj** [comment]
Only solved the conflict but did not change anything else

File: docs/rules/attributes-order.md [hunk 2]
```diff
@@ -4,7 +4,7 @@
 
 ## :book: Rule Details
 
-This rule aims to enforce ordering of component attributes. The default order is specified in the [Vue styleguide](https://vuejs.org/v2/style-guide/#Element-attribute-order-recommended) and is:
+This rule aims to enfore ordering of component attributes. The default order is specified in the [Vue styleguide](https://vuejs.org/v2/style-guide/#Element-attribute-order-recommended) and is:
```
Comment (chrisvfritz) [inline docs/rules/attributes-order.md]: Looks like a typo here.

**Pull request merged**